src/search/style.scss

```diff
@@ -22,11 +22,11 @@
 			border: var( --stc-search-input--focus-border, none );
 		}
 
-		&:hover,
-		&:active,
-		&:focus {
-			box-shadow: 0px 2px 6px -1px rgb(0 0 0 / 6%), 0px 4px 8px -1px rgb(0 0 0 / 10%);
-		}
+		// &:hover,
```

Comment:
Do we need commented code?

Reply:
Please get the necessary inputrs from the designer and implement the changes

Reply:
added it back with new value.